### PR TITLE
Create Categories Cards

### PR DESCRIPTION
**Pull request by krazemon**

### CODE REVIEW

File: api/models/Transaction.js [hunk 1]
```diff
@@ -149,17 +150,92 @@ TransactionSchema.statics.findMostRecent = function(timeStamp) {
...
+            { $lt: ['$amount', 0] },
+            {$subtract: [0, '$amount'] },
+            '$amount'
+          ]
+        }
+      }
+    },
+    { $group: { _id: null, total: { $sum: '$amount'} } },
+    { $sort: { total: 1 } },
+    { $limit: 6 }
```
Comment (ritmatter) [inline api/models/Transaction.js]: Why the change here? This should only return one value, not many.

File: api/models/Transaction.js [hunk 2]
```diff
@@ -149,17 +150,92 @@ TransactionSchema.statics.findMostRecent = function(timeStamp) {
...
+                1
+              ]
+            },
+          "to.id": 1,
+          "to.name": 1
+          }
+        },
+    {
+      $group: {
+        _id: {id: "$to.id", name: "$to.name"},
```
Comment (ritmatter) [inline api/models/Transaction.js]: Any reason you prefer to package the variables like this? Not attached to any particular way to do it, just asking.

File: js/components/CategoriesCards.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+          </div>
+        );
+      }
+    }
+
+    return (
+      <div className="welcomeCards panel panel-default">
+        <div className="panel-heading">
+          <h1 className="panel-title">
+            Welcome to Repcoin! Here are some of the most popular categories
```
Comment (ritmatter) [inline js/components/CategoriesCards.jsx]: What does this look like? I think it should be pretty large.

File: js/components/CategoryCard.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      if (user.picture) {
+       img = (
+         <img className="feed-item-img" src={user.picture.url}>
+         </img>
+       );
+      }
+
+      if (user.about) {
+        about = (
+          <p>{user.about.substr(0,50)}...</p>
```
Comment (ritmatter) [inline js/components/CategoryCard.jsx]: cool

File: js/stores/AuthStore.js [hunk 5]
```diff
@@ -99,6 +99,15 @@ var AuthStore = assign({}, EventEmitter.prototype, {
     return _passwordResetStatus.error;
   },
 
+  isNewby: function() {
```
Comment (ritmatter) [inline js/stores/AuthStore.js]: Do we want everyone to be a newby? Maybe I'm not reading this properly

File: api/models/Transaction.js (hunk 1 above)
Comment (krazemon) [inline api/models/Transaction.js]: I don't know how that happened, my bad.

File: js/stores/AuthStore.js (hunk 5 above)
Comment (krazemon) [inline js/stores/AuthStore.js]: Fuck forgot to remove comment.

File: js/components/CategoriesCards.jsx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+          <div>
+            <CategoryCard name={category.name} users={category.users}
+              currentUser={this.state.currentUser}/>
+          </div>
+        );
+      }
+    }
+
+    return (
+      <div className="welcomeCards panel panel-default">
```
Comment (ritmatter) [inline js/components/CategoriesCards.jsx]: Can we make this bigger and colorful?

File: js/components/CategoryCard.jsx [hunk 7]
```diff
@@ -0,0 +1,55 @@
+var React = require('react');
+
+var Link = require('react-router').Link;
+
+var MiniInvestButton = require('./MiniInvestButton.jsx');
+
+var CategoryCard = React.createClass({
```
Comment (ritmatter) [inline js/components/CategoryCard.jsx]: Some of these cards bleed over the edge of the box. I also think this could be styled a little more, but I'm okay with pushing an iteration 0. But, I do think the users bleeding over the boxes is pretty strange.

File: api/models/Transaction.js (hunk 2 above)
Comment (krazemon) [inline api/models/Transaction.js]: It keeps the records distinct on both fields, also that's how they did it here: http://docs.mongodb.org/manual/reference/operator/aggregation/group/ .

**Comment by ritmatter** [comment]
LGTM!
